html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,12 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
-    //ここからコードを書く
-    echo $i;
+    if ( $i % 3 == 0 &&  $i %5 == 0){
```

Comment:
半角スペースの入れ方が雑なので、
VSCodeの拡張機能のphp intelephenseで自動整形をしてください。条件式の半角スペースが2回連続している個所は1回で十分です。

また、厳密な比較演算子を使った方がよいです。

まとめるとこうなります。

```suggestion
    if ($i % 3 === 0 && $i % 5 === 0) {
```